tips_for_beginners.md

```diff
@@ -43,3 +44,19 @@ Visual Studio Code, Postman, Docker Desktop, three terminal windows
 8. If the difference is not okay, make changes to your files and follow steps 2 to 5 again.
 9. When making a pull request, assign Assignees, reviewers, and labels.
 
+### Run test cases
```

Comment:
from a shell, we can simply connect to the `fastapi` container and run `pytest`

```
$ docker exec -it fastapi pytest
```

Reply:
wow...!! That runs whole tests!! Thank you!!